Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Messagebox/Classes/InternalKryptonMessageBoxExtended.cs

```diff
@@ -237,8 +237,8 @@ private void InitialiseComponent()
         private readonly ExtendedMessageBoxIcon _icon;
         private readonly MessageBoxDefaultButton _defaultButton;
         private AnchorStyles _optionalCheckBoxAnchor;
-        private MessageBoxOptions _options; // TODO: What is this used for ?
-        private bool _fade, _showOptionalCheckBox, _showCopyButton, _hasTimedOut;
+        private MessageBoxOptions _options;
```

Comment:
Why was the TODO removed ? It has not been impleented / described !

Reply:
The `MessageBoxOptions` seems to deal with RTL/LTR text layouts. Not sure if this works?

Reply:
The Winform Messagebox takes one of these. The TODO is a reminder to implement in KMB, and could be extended / augmented / replaced to take the parameters for all these extra things you are getting a messageBox to implement; rather than having an ever ending number of parameters.

Reply:
Reinstated this